src/main/scala/com/advancedtelematic/campaigner/db/Repository.scala

```diff
@@ -298,14 +298,19 @@ protected class UpdateRepository()(implicit db: Database, ec: ExecutionContext)
     findByExternalIdsAction(ns, Seq(id)).failIfNotSingle(Errors.MissingExternalUpdate(id))
   }
 
-  def all(ns: Namespace, sortBy: SortBy = SortBy.Name, nameContains: Option[String] = None): Future[Seq[Update]] = db.run {
```

Comment:
I would prefer to parametrize the `all` function on update type

Reply:
Sure, I've updated this on #79 now.